src/app/components/PageLayout/Logotype.tsx

```diff
@@ -17,30 +16,22 @@ export const HomePageLink: FC<LogotypeProps> = ({ color, showText }) => {
   const { t } = useTranslation()
 
   return (
-    <Link
-      aria-label={t('home.link')}
-      to="/"
-      component={RouterLink}
-      sx={{ display: 'inline-flex', textDecoration: 'none' }}
-    >
-      <Logotype color={color} showText={showText} />
+    <Link asChild aria-label={t('home.link')}>
+      <RouterLink to="/">
+        <Logotype color={color} showText={showText} />
+      </RouterLink>
     </Link>
   )
 }
 
 export const Logotype: FC<LogotypeProps> = ({ color, showText }) => {
-  const theme = useTheme()
   const { isMobile } = useScreenSize()
   const oasisLogoSize = isMobile ? 32 : 40
   const logoSize = !showText ? { height: oasisLogoSize, width: oasisLogoSize } : { height: 40, width: 214 }
 
   return (
-    <Box
-      sx={{
-        color: color || theme.palette.layout.main,
-      }}
-    >
+    <div className={cn('flex items-center', !color && 'text-primary')} style={color ? { color } : undefined}>
```

Comment:
I guess it's easier when handling color via classname or inline styles not both. other than that, we can kill color prop and just have 
`<div className="flex items-center text-primary">` , right?